Aptoide/aptoide-client-v8
# feature/ASV-3-RemoveSpotAndShare

**What does this PR do?**
Removed Spot&Share model classes, client-server lib (module), strings and all associated resources.

**Database changed?**
No

**Where should the reviewer start?**
HomeFragment, where the Spot&Share entry was.
AppViewFragment where through the share button the user could share an app via spot&share.
InstalledAppDisplayable from where the user could share an installed app via Spot&Share.
LoaderLayoutHandler (no network view) from where the user could share an app via Spot&Share.

**How should this be manually tested?**
Open drawer and check for Spot&Share entry.
Check for the aforementioned share buttons (AppViewFragment, InstalledAppDisplayable and LoaderLayoutHandler).

**What are the relevant tickets?**

Associated story : [ASV-3](https://aptoide.atlassian.net/browse/ASV-3)
  Tickets related to this pull-request: [ASV-107](https://aptoide.atlassian.net/browse/ASV-107)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 
No.


**Code Review Checklist**
